src/core/view_manager.cpp

```diff
@@ -861,6 +852,56 @@ void ViewManager::redirect_join_attempt(DerechoSST& gmsSST) {
             curr_view->member_ips_and_ports[rank_of_leader]));
 }
 
+void ViewManager::process_new_sockets() {
+    tcp::socket client_socket;
+    {
+        auto pending_new_sockets_locked = pending_new_sockets.locked();
+        client_socket = std::move(pending_new_sockets_locked.access.front());
+        pending_new_sockets_locked.access.pop_front();
+    }
+    dbg_default_info("Background thread got a connection from {}", client_socket.get_remote_ip());
```

Comment:
This message was already printed by the background thread, so it's redundant here.